tools/src/libtools/cubicspline.cc

```diff
@@ -110,6 +110,8 @@ void CubicSpline::Fit(const Eigen::VectorXd &x, const Eigen::VectorXd &y) {
   // now do a constrained qr solve
   Eigen::VectorXd sol = linalg_constrained_qrsolve(A, y, B);
 
+#ifndef __INTEL_LLVM_COMPILER
+  /* isnan/isinf is always false in fast floating point modes on intel */
```

Comment:
@JensWehner is there a way to test against fast floating point mode?

Reply:
Good question. It is a compiler option. We could run a small test example to see if ffp is used

Reply:
yeah, or just disable the warning here...